### ISSUE TITLE
OGR Error when converting OSM scratch layer to permanent

### ISSUE THREAD

**Issue by mjhorning** [qgis/QGIS#30768]
I am using the tutorial to practice getting data from OSM. I extracted a point layer  then attaempted to make it permanent and received the following error message:

"Could not make temporary scratch layer permanent.
Error: Creation of field name:zea failed (OGR error: Can't create field name_zea in Shape DBF file, reason unknown.)"

**Describe the bug**

**How to Reproduce**

**QGIS and OS versions**

**Additional context**

**Comment by gioman** [comment]
please add the *exact* (detailed) steps you are following. Also the QGIS version you are using (that is mandatory when filing a ticket).

**Comment by mjhorning** [comment]
Apologies
QGIS Version 3.4.9-Madeira
I used QuickOSM to extract the data from OSM. I clicked on the scratch layer icon to pull up Save Scratch Layer dialog. Format: ESRI Shapefile; Encoding: UTF-8; Resize: No.

**Comment by gioman** [comment]
it the error from the plugin? I still don't see the detailed list of steps to follow. If you find more convenient you can attach a screencast.

**Comment by mjhorning** [comment]
These are the steps I followed, directly from the tutorial in the Training Manual (https://docs.qgis.org/3.4/en/docs/training_manual/foreword/preparing_data.html)

"Open a new QGIS project
Select Layer ‣ Data Source Manager to open the Data Source Manager dialog
In the Browser tab, expand the XYZ Tiles drop-down menu and double-click the OpenStreetMap item.
Close the Data Source Manager dialog
Move to the area you’d like to use as study area
Go to Plugins ‣ Manage/Install Plugins…
In the All tab, type QuickOSM in the search box
Select the QuickOSM plugin, press Install Plugin and then Close the dialog.
Execute the new plugin from Vector ‣ QuickOSM ‣ QuickOSM… menu
In the Quick query tab, select place in the Key drop-down menu
Select Canvas Extent in the next drop-down menu
Press Run query
Click the icon next to the place layer to open the Save Scratch Layer dialog.
Select the ESRI Shapefile format
Use the … button to browse to the exercise_data/shapefile/ folder and save the file as places.shp.
Press OK"

I then received the error message. Each time I have tried it a different field name causes the error. That is all I know. I have no idea if the error is traceable to the plugin, the scale of the data I extracted, etc. I am not a programmer or engineer. I am just trying to report a problem I had. Sorry.
